src/components/MutationsTable.tsx

```diff
@@ -0,0 +1,257 @@
+                <td key={`${name}/${i}`}>{name}</td>
+              ))}
+            </tr>
+          </thead>
+          <tbody>
+            {mutations &&
+              mutations.map(item => (
+                <tr key={item.name}>
+                  <td>
+                    {renderNameCell(
+                      item.query,
+                      locationSelector,
+                      item.name,
+                      true,
+                      dateRangeSelector,
+                      baselineVariant
+                    )}
+                  </td>
+                  {mutationColumnsProperties.map(field => {
+                    return (
+                      <td key={field.label}>
+                        {mutationType === 'aa'
+                          ? sortMutationList(
+                              'aa',
+                              pullAll(
+                                getNames(filterMutations(item.aaMutations, field)),
+                                getNames(filterBaselineMutations(baselineData.aaMutations))
+                              )
+                            ).join(', ')
+                          : sortMutationList(
+                              'nuc',
+                              pullAll(
+                                getNames(filterMutations(item.nucMutations, field)),
+                                getNames(filterBaselineMutations(baselineData.nucMutations))
+                              )
+                            ).join(', ')}
+                      </td>
+                    );
+                  })}
+
+                  {mutationColumnsProperties.map(field => {
+                    return (
+                      <td key={field.min}>
+                        {mutationType === 'aa'
+                          ? sortMutationList(
+                              'aa',
+                              compareToBaseline(
+                                field.min,
+                                item.aaMutations,
+                                filterBaselineMutations(baselineData.aaMutations)
+                              )
+                            ).join(', ')
+                          : sortMutationList(
+                              'nuc',
+                              compareToBaseline(
+                                field.min,
+                                item.nucMutations,
+                                filterBaselineMutations(baselineData.nucMutations)
+                              )
+                            ).join(', ')}
+                      </td>
+                    );
+                  })}
+                </tr>
+              ))}
+          </tbody>
+        </table>
+      </div>
+    </>
+  );
+};
+
+const getNames = (mutations: MutationProportionEntry[]) => {
+  return mutations.map(item => item.mutation);
+};
+
+function commonElements<T>(arr1: T[], arr2: T[]): T[] {
+  return arr1.filter(function (element) {
+    return arr2.indexOf(element) !== -1;
```

Comment:
Out of interest: Is `indexOf` better than `includes`? `includes` would make the intention clearer.

Reply:
`arr1.filter(arr2.includes)` looks dangerous because `filter` does not just pass the element to the callback function but actually `(element, index, array)`, and `includes` accepts `fromIndex` as its second argument. I.e., `filter's index` will be used as `includes' fromIndex`